addons/mail/static/src/components/link_preview_delete_confirm/link_preview_delete_confirm.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.LinkPreviewDeleteConfirm" owl="1">
```

Comment:
it's actually in the spec that we need a confirm for removing preview? it seems a bit overkill